### PR TITLE
Initial version of metaform members and groups

### LINKED ISSUE #45: Metaform members
As an administrator I would like to have support for creating, listing, finding, updating and deleting metaform members and metaform member groups so I could associate people with forms

### PR DESCRIPTION
**Pull request by anttileppa**
Resolves #45

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -44,7 +53,10 @@ paths:
     post:
       operationId: createMetaform
       summary: create new Metaform
-      description: Creates new Metaform
+      description: |
+        Creates new Metaform
+        
+        Method is allowed for system administrators and users with metaform administrators
```
Comment (EVP1989) [inline swagger.yaml:59]: This sounds odd, should it be "users with metaform admin rights"?

Comment (EVP1989) [reply swagger.yaml:59]: Same question for other similar descriptions.

File: swagger.yaml [hunk 2]
```diff
@@ -679,7 +717,10 @@ paths:
     post:
       operationId: createReply
       summary: create new form reply
-      description: Creates new form reply
+      description: |
+        Creates new form reply
+        
+        For public forms that have allowAnonymous flag on, method requires that user is logged in for forms without the allowAnonymous flag, user needs to have "user" role in order to post the reply.
```
Comment (EVP1989) [inline swagger.yaml:723]: Description is bit confusing. Is "user" related to Keycloak?

File: swagger.yaml [hunk 3]
```diff
@@ -2193,6 +2787,59 @@ components:
               items:
                 $ref: "#/components/schemas/MetaformScript"
 
+    MetaformMember:
+      type: object
+      description: Form member or manager user.
```
Comment (EVP1989) [inline swagger.yaml:2792]: Is every member basicly also manager or admin?

File: swagger.yaml (hunk 2 above)
Comment (Chi-Chu319) [inline swagger.yaml:723]:
```suggestion
        For public forms that have allowAnonymous flag on, method requires that user is logged in. For forms without the allowAnonymous flag, user needs to have "user" role in order to post the reply.
```

**Review by Chi-Chu319 — APPROVED** [review]

**Pull request merged**